public-ui/src/app/utils/db-constants.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class DbConstantsService {
```

Comment:
Does this need to be a service, or could you just be putting these values into some static utility package and referencing them directly? The only reason I can think you might want this to be a service would be if you expect you'll have a strong need to stub out these constant values for testing. If so, that's fine

Reply:
I was thinking more like if we wanted them to be dynamic constants from a database or change if this is used in workbench etc. And we may add more variables that are dynamic things from the database.

Reply:
We have a similar concept in workbench which we call  the "server config". Optional: could call this DbConfigService to match that.